prow/cluster/jobs/istio/proxy/istio.proxy.master.yaml

```diff
@@ -47,13 +47,30 @@ presubmits:
   istio/proxy:
 
   - name: proxy-presubmit
-    context: prow/proxy-presubmit.sh
+    context: "prow/proxy-presubmit"
     branches: *branch_spec
     always_run: true
+    decorate: true
+    path_alias: istio.io/proxy
```

Comment:
Add `timeout: 60m` to match the timeout previously provided to bootstrap.

Reply:
Done

Reply:
Where's the timeout?

Reply:
1 line down.